kmm/kmm-feed/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeed/FeedViewModel.kt

```diff
@@ -103,5 +132,6 @@ class FeedViewModel(
         val blockingError: Throwable? = null,
         val nonBlockingError: Throwable? = null,
         val sectionedFeed: SectionedFeed? = null,
+        val searchQuery: String = ""
```

Comment:
I’m thinking maybe make this nullable with null as default? We’re not allowing empty queries on the debounce so probably shouldn’t be empty in the state either.

Reply:
done